packages/grid/_modules_/grid/locales/frFR.ts

```diff
@@ -0,0 +1,81 @@
+import { getGridLocalization, Localization } from '../utils';
+
+export const frFR: Localization = getGridLocalization({
+  // Root
+  rootGridLabel: 'grid',
+  noRowsLabel: 'Pas de résultats',
+  errorOverlayDefaultLabel: 'Une erreur est apparue.',
+
+  // Density selector toolbar button text
+  toolbarDensity: 'Densité',
+  toolbarDensityLabel: 'Densité',
+  toolbarDensityCompact: 'Compact',
+  toolbarDensityStandard: 'Standard',
+  toolbarDensityComfortable: 'Confortable',
+
+  // Columns selector toolbar button text
+  toolbarColumns: 'Colonnes',
+  toolbarColumnsLabel: 'Choisir les colonnes',
+
+  // Filters toolbar button text
+  toolbarFilters: 'Filtres',
+  toolbarFiltersLabel: 'Afficher les filtres',
+  toolbarFiltersTooltipHide: 'Cacher les filtres',
+  toolbarFiltersTooltipShow: 'Afficher les filtres',
+  toolbarFiltersTooltipActive: (count) => `${count} filtre(s) actif(s)`,
```

Comment:
why not check the count value and put the 's' accordingly?

Reply:
> why not

Because we should start doing it first in the enUS locale?

Reply:
I'm adding it